setup.py

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Please don't delete all the __future__ imports though - just remove the unicode_literals.  Sure, the others are probably overkill for setup.py, but they are good practice anyway.